Description: The "-race" flag for golang is available only for amd64, arm64, ppc64el. Since this is being used only in a test, it can be dropped
Author: Nilesh Patra <nilesh@debian.org>
Bug-Debian: https://bugs.debian.org/988101
Last-Update: 2021-05-08
--- a/suite/suite_test.go
+++ b/suite/suite_test.go
@@ -552,10 +552,10 @@
 }
 func TestFailfastSuiteFailFastOn(t *testing.T) {
 	// To test this with failfast on (and isolated from other intended test failures in our test suite) we launch it in its own process
-	cmd := exec.Command("go", "test", "-v", "-race", "-run", "TestFailfastSuite", "-failfast")
+	cmd := exec.Command("go", "test", "-v", "-run", "TestFailfastSuite", "-failfast")
 	var out bytes.Buffer
 	cmd.Stdout = &out
-	t.Log("Running go test -v -race -run TestFailfastSuite -failfast")
+	t.Log("Running go test -v -run TestFailfastSuite -failfast")
 	err := cmd.Run()
 	t.Log(out.String())
 	if err != nil {
